modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -322,14 +322,12 @@ inline std::vector<int64_t> toORT(const cv::MatSize &sz) {
 inline void preprocess(const cv::Mat& src,
                        const cv::gimpl::onnx::TensorInfo& ti,
                              cv::Mat& dst) {
-    GAPI_Assert(src.depth() == CV_32F || src.depth() == CV_8U);
     // CNN input type
     const auto type = toCV(ti.type);
-    if (src.depth() == CV_32F) {
+    if (src.depth() != CV_8U) {
         // Just pass the tensor as-is.
         // No layout or dimension transformations done here!
         // TODO: This needs to be aligned across all NN backends.
-        GAPI_Assert(type == CV_32F && "Only 32F model input is supported for 32F input data");
```

Comment:
What about this: https://github.com/opencv/opencv/blob/f8ffdae050bc329a1101fb4a05279ad417dc1d55/modules/gapi/src/backends/onnx/gonnxbackend.cpp#L335? Still refers only to `FP32`, can be at least not `U8`?

Reply:
you're right, will change this now

Reply:
Looks not changed